app/assets/javascripts/discourse/app/models/draft.js

```diff
@@ -23,11 +23,17 @@ Draft.reopenClass({
     return current;
   },
 
-  save(key, sequence, data, clientId) {
+  save(key, sequence, data, clientId, forceSave) {
```

Comment:
I am not a fan of adding boolean parameters to the end of function signatures like this, because it's not usually clear at the call site what the parameter represents.

In this case we should prefer an `opts` params which is an object.  Then the call site is easier to understand: `{ forceSave: true }`